pombola/south_africa/templates/info/_blog_sidebar.html

```diff
@@ -14,7 +14,7 @@
       {% endfor %}
     </ul>
 
-    {% if popular_posts.exists %}
+    {% if some_popular_posts %}
```

Comment:
DO you want to have an `{% empty %}` section here too, now that it's theoretically possible to get here but not display anything? (If all the blog posts are old, for example.)